erpnext_com/www/docs/user/manual/en/education/navigating-the-portal.md

```diff
@@ -0,0 +1,67 @@
+# Navigating the Portal
+
+## Home Page
+The portal is hosted on `/lms` route. (Example. `hogwarts.erpnext.com/lms`) Here all the programs are shown in the form of cards. Each card is clickable and navigates to the corresponding program or course or topic or content. The portal is accessible even if a student has not logged in, however the content (Articles, Videos, etc) can oly be accessible after login and enrolling in the program.
+
+<img class="screenshot" alt="LMS Home" src="{{docs_base_url}}/assets/img/education/lms/home.png">
+
+## Porgram Page
+On the program page, the student can see the description as well as the list of courses in the form of cards. If the student is not enrolled, a button to enroll will be shown. This will happen only if self enroll is enabled in the program doc, if not then the program wont be visible to the student at all.
+
+After enrolling the status of each course is added to the card footer. This status is updated based on the students activity on the portal.
+
+<img class="screenshot" alt="Portal Program Enrolled" src="{{docs_base_url}}/assets/img/education/lms/program-enrolled.png">
+<img class="screenshot" alt="Portal Program Progress" src="{{docs_base_url}}/assets/img/education/lms/program-progress.png">
+
+---
+
+## Course Page
+Similar to the program page, it lists all the topics, as well as the status of each topic. Clicking on any card will navigate to the topic page. A list of contents is shown in each topic card, clicking on any list item will navigate to that content.
+
+<img class="screenshot" alt="Portal Course" src="{{docs_base_url}}/assets/img/education/lms/completed_course.png">
+
+---
+
+## Topic Page
+The topic page lists all the content as well as their completion status. Clicking on any card will navigate to the content Page
+
+<img class="screenshot" alt="Portal Course" src="{{docs_base_url}}/assets/img/education/lms/topic.png">
+
+---
+
+## Content Pages
+
+Each type of content has it's own view and a common navigation based on the content type.
+
+<img alt="Portal Contents" src="{{docs_base_url}}/assets/img/education/lms/content.png">
+
+When a student visits particular content page, he/she can navigate using buttons at the bottom of the content.
+
+> Note: The activity of the student is only recorded after the student clicks on the Next button.
+
+<img class="screenshot" alt="Portal Contents" src="{{docs_base_url}}/assets/img/education/lms/content-navigation.png">
+
+### Navigating Quizzes
+
+In case of a quiz, the student has to first submit the quiz, following which the result is computed and displayed, after which the student can navigate to the next content.
+
+<img class="screenshot" alt="Quiz Pass" src="{{docs_base_url}}/assets/img/education/lms/quiz-pass.png">
```

Comment:
even here the radio buttons seems glitchy on the right side
not to mention the turfs on either side of the page